portfolio/src/main/webapp/script.js

```diff
@@ -13,7 +13,11 @@
 // limitations under the License.
 
 
-
+async function getPhrase() {
+  const response = await fetch('/data');
+  const phrase = await response.text();
```

Comment:
Please use let for local variables that get overwritten or changed. Const is generally for variables that get assigned once and never again.

https://google.github.io/styleguide/jsguide.html#features-use-const-and-let

Reply:
I'll keep that in mind, but for this implementation I did not change the values assigned to response and phrase after initializing them